Sources/RichTextKit/Component/RichTextViewComponent+Attributes.swift

```diff
@@ -32,24 +33,46 @@ public extension RichTextViewComponent {
         currentRichTextAttributes[attribute] as? Value
     }
 
+    /// Get the current value of a rich text attribute.
+    func currentCustomRichTextAttribute(
```

Comment:
Is this used? I find it a bit confusing how we now have current, typing and currentTyping naming in the codebase. I'll check if I can unify it. I think `richTextTypingAttribute` may be a good naming to make the "rich text" name space obvious, and replace "Current" with "Typing" as a naming convention.

But then we have to be consistent.